### ISSUE TITLE
Importing/Exporting secret keys

### ISSUE THREAD

**Issue by RangerMauve** [cliqz-oss/dat-webext#25]
It'd be nice if there was a way to copy or paste secret keys managed by the extension.

This is a pretty dangerous feature, but it'd be nice to have for backing up keys in case I want to transfer devices or clear storage.

**Comment by sammacbeth** [comment]
Yes. I think this could be handled by a library view, giving details on the Dats you have locally in your browser. While I'm working on the core protocol functionality I won't have time to look at this, but if anyone wants to jump on this, feel free!
